db/index.js

```diff
@@ -6,6 +6,8 @@ import Postgrator from 'postgrator'
 
 // re-export types
 /** @typedef {import('./typings.js').PgPools} PgPools */
+/** @typedef {import('./typings.js').PgPoolStats} PgPoolStats */
+/** @typedef {import('./typings.js').PgPoolEvaluate} PgPoolEvaluate */
 /** @typedef {import('./typings.js').Queryable} Queryable */
```

Comment:
FWIW, with the latest TypeScript version, we should be able to simplify this block as follows:

```js
/** @import { PgPools, PgPoolStats, PgPoolEvaluate, Queryable } from './typings.js' */
```

See https://devblogs.microsoft.com/typescript/announcing-typescript-5-5/#the-jsdoc-import-tag

Reply:
Nevermind. Such an import statement does not re-export the types.